treeherder/log_parser/parsers.py

```diff
@@ -319,6 +319,11 @@ def parse_line(self, line, lineno):
                 title, value = line.split("<br/>", 1)
                 artifact["title"] = title
                 artifact["value"] = value
+            # and split TinderboxPrint or similar long lines
+            elif "href" in line and "title" in line:
```

Comment:
Ok, I've pushed that change below. I've re-tested locally with comm-central push `2540c39cc958` from Aug 28th and parsing appears to be correct for Linux/OSX/Win platforms. I also ingested a generic push from mozilla-central, and with the one **[job](https://treeherder.mozilla.org/#/jobs?repo=mozilla-central&revision=7a7ba250bb2f5a7cc7acf4b97145425c5292e894&selectedJob=5209668)** I could get ingested and parsed locally, the job details appear to be correct. I'm not sure about the performance impact of the windows `^M` return suppression I had to add for handle_data, but I assume it's no worse than constantly testing for a platform and then processing.